src/adapter/adapter.ts

```diff
@@ -151,7 +151,7 @@ abstract class Adapter extends events.EventEmitter {
      */
 
     public abstract sendZclFrameToEndpoint(
-        networkAddress: number, endpoint: number, zclFrame: ZclFrame, timeout: number,
+        networkAddress: number, endpoint: number, zclFrame: ZclFrame, timeout: number, srcEndpoint?: number
```

Comment:
For consistency purposes, please add the `srcEndpoint` to every `sendXXX` method.

Reply:
Added